src/ontology/edits/oeo-model.omn

```diff
@@ -4008,6 +4008,19 @@ Pull: https://github.com/OpenEnergyPlatform/ontology/pull/2091"
         OEO_00020166
     
     
+Class: OEO_00390105
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A delayed transition scenario is an explorative scenario whose intermediate transition indicators are not achieved.",
+        rdfs:label "delayed transition scenario"@en,
+        OEO_00020426 "add:
+Issue: https://github.com/OpenEnergyPlatform/ontology/issues/2101
+Pull Request: https://github.com/OpenEnergyPlatform/ontology/pull/2116"
+    
+    SubClassOf: 
+        OEO_00000364
```

Comment:
```suggestion
        OEO_00020248
```

Reply:
https://openenergyplatform.org/ontology/oeo/OEO_00020248 is explorative scenario. Before it was a direct subclass of scenario.